modules/node-shared/src/main/scala/org/tessellation/node/shared/domain/statechannel/FeeCalculator.scala

```diff
@@ -28,6 +29,17 @@ case class FeeCalculatorConfig(
   proWeight: NonNegBigDecimal
 )
 
+case class SnapshotFeesInfo(
+  allFeesAddresses: Map[Address, List[Address]],
+  stakingBalance: Balance,
+  maybeOwnerAddress: Option[Address],
+  maybeStakingAddress: Option[Address]
```

Comment:
`maybe` in the name is redundant since the type is self explanatory

Reply:
👍🏼